cppquiz/quiz/templatetags/quiz_extras.py

```diff
@@ -7,12 +7,27 @@
 
 register = template.Library()
 
+def format_paragraph(paragraph_name, paragraph_number, full_text):
```

Comment:
Minor nitpick: I'm a bit unsure about the names here. I think maybe `paragraph_name` should be `section_name`? 

Actually not sure about these names myself, so I asked the Twitterverse, let's see if we get any feedback: https://twitter.com/knatten/status/1132974180104851456